sql/src/test/java/org/apache/druid/sql/calcite/CalciteQueryTest.java

```diff
@@ -174,7 +180,28 @@ public void testSelectNonNumericNumberLiterals() throws Exception
         + " CAST(-1 / 0.0 AS BIGINT),"
         + " CAST(-1 / -0.0 AS BIGINT),"
         + " CAST(0/ 0.0 AS BIGINT)",
-        ImmutableList.of(),
+        ImmutableList.of(
```

Comment:
I think we should call out in the release notes about this behavior change just in case, where using Druid like a calculator is going to do scans with an inline datasource now instead of going down the bindable path (which i think is an improvement, makes things more visible and we want to drop the bindable path at some point in the future)

Reply:
Good point. I added a section for the release note.